docs/doxygen/logging/logging.dox

```diff
@@ -0,0 +1,44 @@
+/**
+@page logging Logging Infrastructure
+@brief This page contains an overview of the logging infrastructure used by the C-SDK Libraries.
+
+<p>
+The C-SDK libraries utilize one of the following 4 logging macro interfaces, each with a different verbosity level:
+- `LogError` for **Error** level
+- `LogWarn` for **Warning** level
+- `LogInfo` for **Info** level
+- `LogDebug` for **Debug** level
+
+**Note: By default, each library contains empty definitions for the above logging macros, thereby generating no code for logging calls.**
```

Comment:
If this is a note, why not make it a `@note`? Also, it doesn't get bolded when I generated the documentation, and just had `**` at the end

Reply:
Interesting observation. Thanks for pointing out.